internal/handlers/pg/pgdb/collections.go

```diff
@@ -160,6 +160,40 @@ func CreateCollection(ctx context.Context, querier pgxtype.Querier, db, collecti
 	}
 }
 
+// CreateCollectionIfNotExist ensures that given FerretDB database / PostgreSQL schema
+// and FerretDB collection / PostgreSQL table exist.
+// If needed, it creates both schema and table.
+//
+// True is returned if table was created.
+func CreateCollectionIfNotExist(ctx context.Context, querier pgxtype.Querier, db, collection string) (bool, error) {
+	exists, err := CollectionExists(ctx, querier, db, collection)
+	if err != nil {
+		return false, lazyerrors.Error(err)
+	}
+
+	if exists {
+		return false, nil
+	}
+
+	// Table (or even schema) does not exist. Try to create it,
+	// but keep in mind that it can be created in concurrent connection.
+
+	if err := CreateDatabase(ctx, querier, db); err != nil && !errors.Is(err, ErrAlreadyExist) {
+		return false, lazyerrors.Error(err)
+	}
+
+	// TODO use a transaction instead of pgPool: https://github.com/FerretDB/FerretDB/issues/866
```

Comment:
Is this comment still actual? It seems that we already use querier here.

Reply:
But we don't use a transation

Reply:
Ok, got it (now I see how transaction is used in update).